app/models/concerns/multi_step_model.rb

```diff
@@ -46,4 +46,7 @@ def method_missing(method_name, *args, &block)
     end
   end
 
-end
\ No newline at end of file
+  def respond_to_missing?(method, *)
+    method =~ /^step(\d+)\?$/ || super
+  end
```

Comment:
Is this method new? if so, I don't think it should be part of this PR. In any case, I'm interested in knowing the rationale behind this :)

Reply:
It's just a recommended practice when you use the method `method_missing` that we were using.